### PR TITLE
basic styling (nav bar, background, all products)

### PR DESCRIPTION
**Pull request by Nievescs20**

### CODE REVIEW

File: client/components/AllProducts.js [hunk 1]
```diff
@@ -16,24 +16,39 @@ class AllProducts extends Component {
...
   }
 
   render() {
-    console.log('these are my props: ', this.props);
+    console.log("these are my props: ", this.props);
     return (
-      <div>
-        <h1>This is the AllProducts component</h1>
+      <div id="all-products">
+        <video
```
Comment (jared-melnyk) [inline client/components/AllProducts.js:27]: @Nievescs20 checkout browser console warnings on this component for "Invalid DOM property". It seems to want all these video properties in camelCase, e.g. playsInLine, controlsList, autoPlay etc

File: client/components/Navbar.js [hunk 2]
```diff
@@ -1,26 +1,37 @@
...
-import { logout } from '../store';
+import React from "react";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import { logout } from "../store";
 
 const Navbar = ({ handleClick, isLoggedIn }) => (
-  <div>
-    <h1>Caffein-E Commerce</h1>
+  <div id="nav-bar">
```
Comment (jared-melnyk) [inline client/components/Navbar.js:7]: @Nievescs20 the nav-bar div now seems to overlay and hide the login & signup elements. Please update CSS positioning to those critical elements still display on the screen.

**Review by jared-melnyk — CHANGES_REQUESTED** [review]
Made a few comments in the file, requesting associated changes prior to merge as critical login/signup HTML elements are hidden based on these style updates.

File: client/components/AllProducts.js (hunk 1 above)
Comment (Nievescs20) [reply client/components/AllProducts.js:27]: not sure about these warnings. if i change property to camelCase the video element breaks

File: client/components/Navbar.js (hunk 2 above)
Comment (Nievescs20) [reply client/components/Navbar.js:7]: this was a simple fix. taken care of

**Review by jared-melnyk — APPROVED** [review]

**Comment by jared-melnyk** [comment]
Looks great, nice job

**Pull request merged**